components/contentblocks/FrontPageHeroBlock.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import HeroFullImage from 'components/home/HeroFullImage';
+import HeroSmallImage from 'components/home/HeroSmallImage';
+
+interface FrontPageHeroProps {
+  id: string;
+  layout: 'small_image' | 'big_image';
```

Comment:
Nitpick: Would `full_width` and `contained` be clearer? An image that could have a height up to 1600px also sounds like a big image :D

Reply:
Absolutely would. Only reason I ended up using this is that we happened to have a legacy Small Image layout option in admin that had no UI effect anymore. I repurposed the unused backend choice for this. 

Small image kiiind of makes sense as in there the image is limited to container width instead of expanding to full browser width :D The height limit for rendition is arbitary as I would hope that no customer would try to put portrait ratio image there...